app/admin/models/Orders_model.php

```diff
@@ -121,8 +121,13 @@ public function scopeListFrontEnd($query, $options = [])
             'customer' => null,
             'location' => null,
             'sort' => 'address_id desc',
+            'search' => '',
+            'from_epoch' => null,
+            'to_epoch' => null,
```

Comment:
I'd name these `startAt` and `endAt` and group them into an array so we can support filtering on other date columns like date_added or date_modified... Something like this (similar to how Square API does it)

```
'dateTimeFilter' => [
    'orderDateTime' => [
        'startAt' => '', 'endAt' => ''
    ]
]
```

Not sure if they should be camelCased or snake_case...

Reply:
Went for snake as the includes are - felt it’s was consistent. 

Had left reservations until you approved the approach :)

Reply:
Actually we use camelCase for pageLimit so makes sense to be consistent with it.
Reservations done now to.

Reply:
Ahh, just seeing your comment... Ignore the snake_case part.